src/framework/preferences/js/PrefsEditor.js

```diff
@@ -371,33 +371,36 @@ var fluid_2_0 = fluid_2_0 || {};
     };
 
     /**
-     * Saves the current model and fires onSave
- */
+     * Sends the prefsEditor.model to the store and fires onSave
+     * @param that: An fluid.prefs.prefsEditor instance
+     * @return the saved model
+     */
     fluid.prefs.prefsEditor.save = function (that) {
         if (!that.model) {  // Don't save a reset model
             return;
         }
 
-        var initialModel = that.initialModel,
-            userSelections = fluid.copy(that.model),
-            changedSelections = {};
-
-        // Only save the changed preferences so the future default value change on preferences can still be correctly merged with changed preferences
-        if (fluid.model.diff(userSelections.preferences, initialModel.preferences)) {
-            delete userSelections.preferences;
-        } else {
-            var stats = {changes: 0, unchanged: 0, changeMap: {}};
+        var modelToSave = fluid.copy(that.model),
+            initialModel = that.initialModel,
+            stats = {changes: 0, unchanged: 0, changeMap: {}},
+            changedPrefs = {};
 
-            fluid.model.diff(userSelections.preferences, fluid.get(that.initialModel, ["preferences"]), stats);
+        // Only save the changed preferences so the future initial model value change on unchanged
+        // preferences can still be correctly merged with changed preferences
+        fluid.model.diff(modelToSave.preferences, fluid.get(initialModel, ["preferences"]), stats);
 
-            fluid.each(stats.changeMap, function (state, path) {
-                fluid.set(changedSelections, ["preferences", path], userSelections.preferences[path]);
+        if (stats.changes === 0) {
+            delete modelToSave.preferences;
+        } else {
+            fluid.each(stats.changeMap, function (state, pref) {
+                fluid.set(changedPrefs, pref, modelToSave.preferences[pref]);
```

Comment:
changeMap generated by fluid.model.diff() only contains the changed preferences. It doesn't list unchanged preferences. Let me know if I didn't understand your question correctly.